languages/javascript/exercises/concept/errors/.docs/instructions.md

```diff
@@ -0,0 +1,26 @@
+You are a developer in a newspaper factory and you are asked to work on a software that prevents  printing machines from overheating. Currently there are technicians that check each hour the temperatures and turn of the machines if they are overheating.
```

Comment:
I like the story. Good job!

Reply:
It may not be very funny, but I think machines are a good example of things that can go wrong :)